components/Op/OpAdd.js

```diff
@@ -41,10 +51,10 @@ export const OpAddOfferBtn = () => {
 const OpAdd = ({ roles }) => {
   if (!roles.length) return null
   return (
-    <>
+    <OppAddButtons>
       {(roles.includes(Role.OPPORTUNITY_PROVIDER)) && <><OpAddAskBtn /> &nbsp; </>}
```

Comment:
Can we remove the &nbsp; and use the margins in the style above now?

Reply:
yep, done.